src/main/java/bio/terra/stairway/FlightMap.java

```diff
@@ -110,8 +109,23 @@ public void fromJson(String json) {
     }
   }
 
+  // Truncate working map to only print first 500 characters
   @Override
   public String toString() {
-    return new ToStringBuilder(this, ToStringStyle.JSON_STYLE).append("map", map).toString();
+    int truncateLength = 500;
+    StringBuilder sb = new StringBuilder("{");
+    sb.append(
+        map.entrySet().stream()
+            .map(
+                set -> {
+                  String valString = set.getValue().toString();
+                  return String.join(
```

Comment:
It wouldn't have occurred to me to use `String.join()` to join two things, is there a reason to use this over `+`? Either way is OK with me.

Reply:
Is it probably overkill. I don't really have a preference - I'll switch it back.